scalanet/src/io/iohk/scalanet/peergroup/dynamictls/DynamicTLSPeerGroupUtils.scala

```diff
@@ -10,11 +10,11 @@ import io.netty.handler.ssl.{ClientAuth, SslContext, SslContextBuilder}
 import javax.net.ssl._
 import scodec.bits.BitVector
 
-import scala.collection.JavaConverters._
+import java.util.Arrays
 
 private[scalanet] object DynamicTLSPeerGroupUtils {
   // supporting only ciphers which are used in TLS 1.3
-  val supportedCipherSuites: Seq[String] = Seq(
+  val supportedCipherSuites: java.lang.Iterable[String] = Arrays.asList(
```

Comment:
```suggestion
  private val supportedCipherSuites: java.lang.Iterable[String] = Arrays.asList(
```
If it's optimised for Java interop it's better to limit its visibility.